### ISSUE TITLE
CG/2014-Nachklausur, Aufgabe 7a (II)

### ISSUE THREAD

**Issue by pjungeblut** [MartinThoma/KIT-Musterloesungen#34]
Ich glaube zur Berechnung von s müsste man ein Minus hinzufügen:

s = -r_x/2r_z + 1/2

Vgl. Kapitel 4, Folie 110.

**Comment by matthiasplappert** [comment]
Nein, die Musterlösung ist richtig, wie einfaches Nachrechnen zeigt:

`0 + t * r = (x, y, 1)` mit `r = (-1, 1, 2)`. Offensichtlich ist `t = 1/2`. Damit folgt für den Schnittpunkt mit der z-Ebene: `s = 1/2 * (-1, 1, 2) = (-1/2, 1/2, 1)`. Normalisieren auf `[0,1]` ergibt dann die Lösung (wie in der Musterlösung auch) von `uv = (1/4, 3/4)`.

**Comment by jpaczia** [comment]
Also berechnet sich s bzw t mit (1/2) + (r_j)/(2*r_i), wobei r_i die Komponente ist, deren Betrag am größten ist und r_j die beiden anderen? Für s wird dann bei r_j die Komponete eingesetzt die im Vektor weiter "oben" steht?
